pkgs/top-level/python-packages.nix

```diff
@@ -28490,15 +28490,16 @@ in modules // {
 
   mps-youtube = buildPythonPackage rec {
     name = "mps-youtube-${version}";
-    version = "0.2.6";
+    version = "0.2.7.1";
 
     disabled = (!isPy3k);
+    doCheck = false;
```

Comment:
if you disable the tests, please include a comment in the expression stating why.